### PR TITLE
Split fever & greader into two separate pages.

### PR DESCRIPTION
**Pull request by Stunkymonkey**
Changes proposed in this pull request:

- add newsflash to fever-api clients
- split APIs into two separate pages.

Pull request checklist:
- [x] documentation updated

### CODE REVIEW

**Comment by Alkarex** [comment]
Thanks @Stunkymonkey !
Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/edge/CREDITS.md
I need to try NewsFlash again.
Do you believe the line for NewsFlash could be updated in the following table?
https://github.com/FreshRSS/FreshRSS/blob/edge/README.md#apis--native-apps

File: docs/en/users/06_Mobile_access.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ This page assumes you have completed the [server setup](../admins/03_Installatio
 	* Every user must define an API password.
 	* The reason for an API-specific password is that it may be used in less safe situations than the main password, and does not grant access to as many things.
 
-The rest of this page is about the Google Reader compatible API.
-See the [page about the Fever compatible API](06_Fever_API.md) for another possibility.
+See the [page about the Google Reader compatible API](../developers/06_GoogleReader_API.md.md) for more details.
+See the [page about the Fever compatible API](../developers/06_Fever_API.md.md) for more details.
```
Comment (Alkarex) [inline docs/en/users/06_Mobile_access.md:11]: `.md.md` broken links, fixed in https://github.com/FreshRSS/FreshRSS/commit/88e47dba80fdca9ad77bac3e1b1e9694c750287c

**Comment by Stunkymonkey** [comment]
i will send a new PR with updates to the `README`

**Comment by Stunkymonkey** [comment]
if you want the "Hacktober" to count for the users please use the label `hacktoberfest-accepted`. Otherwise it does not count.

**Comment by Alkarex** [comment]
> if you want the "Hacktober" to count for the users please use the label `hacktoberfest-accepted`. Otherwise it does not count.

So rename `hacktoberfest` into `hacktoberfest-accepted`, or are two labels needed?

**Comment by Stunkymonkey** [comment]
The `Hacktober` should be used for issues, ... so people could spot what will get accepted. (i am not a huge fan of this practice)
`hacktoberfest-accepted` should be used for PRs and a single label is sufficient.
https://hacktoberfest.digitalocean.com/resources/maintainers

**Comment by Alkarex** [comment]
Ok, I have tried to fix. Let me know if anything else should be done :-)

**Pull request merged**